examples/file-handling-azure-storage/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "prisma-file-handling-azure-storage",
+  "version": "1.0.0",
+  "license": "MIT",
+  "scripts": {
+    "start": "dotenv -- nodemon -x ts-node -e ts,graphql src/index.ts",
+    "debug": "dotenv -- nodemon -x 'ts-node --inspect' -e ts,graphql src/index.ts",
+    "playground": "dotenv -- graphql playground",
+    "bind": "graphql prepare",
+    "build": "rm -rf dist && tsc"
+  },
+  "dependencies": {
```

Comment:
* please use latest versions for all dependencies here, most importantly the `graphql` and `prisma` related ones 🙂 
* the same for the `graphql-cli` and `prisma` dev dependencies